src/app/Http/Controllers/AdminController.php

```diff
@@ -42,4 +45,21 @@ public function redirect()
         // The '/admin' route is not to be used as a page, because it breaks the menu's active state.
         return redirect(backpack_url('dashboard'));
     }
+
+    /**
+     * Show the error.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function errorFrame(Request $request)
+    {
+        $handler = app(ErrorPageHandler::class);
+        $client = app()->make(Flare::class);
+
+        $exception = new $request->exception($request->message, 0, 1, $request->file, $request->line);
```

Comment:
Any chance we can also inject the stack trace here? I'm saying that because:
- As a dev, I'll find the iFrame very confusing if the stack trace is wrong;
- Arguably, the stack trace is what you want, it's the second most important thing for debugging after the error message itself... not everything else Flare provides (that's extra);
- We do have the stack trace as JSON from the previous AJAX request, right? can we inject it into the current error?

If there's not way to do this (basically generate a Flare error on-the-fly including the stack trace)... perhaps we take the exact opposite approach... and generate a CUSTOM page (not a flare error page), with the stack trace and error message? What do you guys think about that, which one would be more useful?

Reply:
Stack trace is now in 🙌